tests/PowerPipe.UnitTests/PipelineTests.cs

```diff
@@ -123,24 +123,3 @@ public async Task IfStep_Succeed(bool predicate)
 }
 
 public record TestPipelineResult;
```

Comment:
I think that this record could be moved to `tests/PowerPipe.UnitTests/Steps/TestPipelineContext.cs`

Reply:
actually it's Ok, I'll fix it here https://github.com/mvSapphire/PowerPipe/pull/10